tests/Tests/Payload/SearchFilesPayloadResponseTest.php

```diff
@@ -39,8 +39,8 @@ protected function assertResponse(array $responseData, PayloadResponseInterface
     {
         $fileResult = $payloadResponse->getResult();
 
-        $this->assertInstanceOf('CL\Slack\Model\FileResult', $fileResult);
-        $this->assertCount(1, $fileResult->getMatches());
+        self::assertInstanceOf('CL\Slack\Model\FileResult', $fileResult);
```

Comment:
For consistency, you might want to use the `...:class` here too instead of the string:

```suggestion
        self::assertInstanceOf(CL\Slack\Model\FileResult::class, $fileResult);
```